packages/core/src/routes/interaction/middleware/koa-interaction-sie.ts

```diff
@@ -24,7 +24,17 @@ export default function koaInteractionSie<StateT, ContextT extends IRouterParamC
   return async (ctx, next) => {
     const signInExperience = await findDefaultSignInExperience();
 
-    ctx.signInExperience = signInExperience;
+    ctx.signInExperience = {
+      ...signInExperience,
+      mfa: {
+        ...signInExperience.mfa,
+        policy:
+          // Fallback deprecated UserControlled policy to PromptAtSignInAndSignUp
+          signInExperience.mfa.policy === MfaPolicy.UserControlled
+            ? MfaPolicy.PromptAtSignInAndSignUp
+            : signInExperience.mfa.policy,
```

Comment:
Shouldn't we fallback to `PromptOnlyAtSignIn` since this is the old way of doing it?

Reply:
The old way is `PromptAtSignInAndSignUp`

Reply:
But the changeset says the old policy is `PromptOnlyAtSignIn`